projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-detail-items/order-detail-items.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="order$ | async as order">
   <ng-container
-    *ngIf="order.consignments?.length || order.unconsignedEntries?.length"
+    *ngIf="order.consignments?.length || order.unconsignedEntries?.length || order.replenishmentOrderCode"
```

Comment:
II wonder what is the purpose of not displaying the order promotions if this condition is not met.  If the backend include promotions in the order data, why not display them?  There must be a reason though.